Odoo.xml

```diff
@@ -57,62 +57,62 @@
       <option name="Python" value="true" />
     </context>
   </template>
-  <template name="odoo_field_integer" value="new_field = fields.Integer(string=&quot;&quot;, required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
+  <template name="odoo_field_integer" value="new_field = fields.Integer(string='', required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
```

Comment:
It is only as visual improvement. A guess the python code more clean when use `single quotes`. 

I can turn back it if you dont like.